integration-cli/docker_cli_daemon_plugins_test.go

```diff
@@ -236,22 +221,6 @@ func (s *DockerDaemonSuite) TestVolumePlugin(c *check.C) {
 	}
 	c.Assert(out, checker.Contains, volName)
 	c.Assert(out, checker.Contains, pName)
-
-	mountPoint, err := s.d.Cmd("volume", "inspect", volName, "--format", "{{.Mountpoint}}")
-	if err != nil {
-		c.Fatalf("Could not inspect volume: %v %s", err, mountPoint)
-	}
-	mountPoint = strings.TrimSpace(mountPoint)
-
-	out, err = s.d.Cmd("run", "--rm", "-v", volName+":"+destDir, "busybox", "touch", destDir+destFile)
```

Comment:
Can we add a way to test using the volume in a container?

Reply:
I'm not really sure what we're testing here.
It looks like we're just testing the the volume data is created in the plugin root, but this isn't really a valid test.
The only thing I can really think to do is to test if using the same volume name yields the same data, which is tested elsewhere.

Do you have an idea?

Reply:
Create data in one container and use it in another? 
We are also missing `propagated-mount`  related tests.

Reply:
We can do that, but then we're testing the plugin and not docker.

Reply:
The way I see it is, we are using a mock plugin to ensure docker is offering the said behavior.

Reply:
Updated the test